core/src/main/java/com/scalar/db/storage/jdbc/RdbEngineMysql.java

```diff
@@ -448,7 +449,19 @@ public TimestampTZColumn parseTimestampTZColumn(ResultSet resultSet, String colu
   }
 
   @Override
-  public Map<String, String> getConnectionProperties() {
+  public Map<String, String> getConnectionProperties(JdbcConfig config) {
+    if (config.getDatabaseConfig().getScanFetchSize() == Integer.MIN_VALUE) {
+      // If the scan fetch size is set to Integer.MIN_VALUE, use the streaming mode.
+      return Collections.emptyMap();
+    }
+
+    // Otherwise, use the cursor fetch mode.
     return Collections.singletonMap("useCursorFetch", "true");
   }
```

Comment:
If a user sets `scalar.db.scan_fetch_size` to `Integer.MIN_VALUE`, the streaming mode is used in MySQL. In that case, we should not set the connection property `useCursorFetch=true`.